### PR TITLE
[fix] add ability to configure timeout

### PR DESCRIPTION
**Pull request by jcrugzz**

### CODE REVIEW

**Comment by Swaagie** [comment]
LGTM

File: index.js [hunk 1]
```diff
@@ -103,6 +103,10 @@ module.exports = function createServers(options, listening) {
         port   = !isNaN(options.http.port) ? +options.http.port : 80, // accepts string or number
         args;
 
+    if (typeof options.http.timeout === 'number') {
+      server.setTimeout(options.http.timeout);
```
Comment (3rd-Eden) [inline index.js]: I'm gonna be the annoying person who complains about code style.. In this ifstatement you use curly braces but in the other one for SSL you don't. YOUR GRINDING MY GEARS HERE SIR.

**Comment by 3rd-Eden** [comment]
"Do you know how to write mocha tests"

**Comment by 3rd-Eden** [comment]
@jcrugzz

**Pull request merged**